platform/darwin/src/MGLLight.h

```diff
@@ -54,72 +63,131 @@ MGL_EXPORT
 @interface MGLLight : NSObject
 
 /**
- `anchor` Whether extruded geometries are lit relative to the map or viewport.
+ Whether extruded geometries are lit relative to the map or viewport.
+ 
+ The default value of this property is an `MGLStyleValue` object containing an
+ `NSValue` object containing `MGLAnchorViewport`.
  
+ You can set this property to an instance of:
+ 
+ * `MGLConstantStyleValue`
+ * `MGLCameraStyleFunction` with an interpolation mode of
+ `MGLInterpolationModeInterval`
+
  This property corresponds to the <a
  href="https://www.mapbox.com/mapbox-gl-js/style-spec/#light-anchor"><code>anchor</code></a>
  light property in the Mapbox Style Specification.
  */
 @property (nonatomic) MGLStyleValue<NSValue *> *anchor;
 
 /**
- Values describing animated transitions to `anchor` property.
- */
-@property (nonatomic) MGLTransition anchorTransition;
-
-
-/**
- Position of the light source relative to lit (extruded) geometries.
+ Position of the light source relative to lit (extruded) geometries, in [r
+ radial coordinate, a azimuthal angle, p polar angle] where r indicates the
```

Comment:
This documentation comment is a good candidate for a Cocoa override, because it discusses an array syntax that isn’t relevant to Objective-C or Swift. The ideal documentation for this property would refer to `MGLLightAnchor` and let the documentation for that symbol describe the rest of the details.